### DISCUSSION TITLE
Experimental syslog-ng packages for Amazon Linux 2023

### DISCUSSION THREAD

**Discussion by czanik** [syslog-ng/syslog-ng#4965] [Announcements]
Last year, I received many requests about syslog-ng for Amazon Linux 2023, but I could not find an easy way to create syslog-ng packages. Recently, however, I found that Fedora Copr supports building packages for Amazon Linux 2023. So, with a little bit of experimentation, I got a cut down version of syslog-ng compiled.

Here I'd like to collect your feedback about syslog-ng Amazon Linux 2023 packaging. For more details, check the related blog at: https://www.syslog-ng.com/community/b/blog/posts/experimental-syslog-ng-packages-for-amazon-linux-2023

**Comment by mexigabacho** [comment]
Is there anyway to bring this back by chance?

**Comment by czanik** [comment]
Yes. No ETA, but I'll update the repo to the latest version when I have some idle time. The fun thing is that according to Copr statistics, this is one of my most used repositories. But I asked the community many times if I should update it, and got no response at all...

**Comment by mexigabacho** [comment]
I completely understand.  I think it's the nature of the package to some
extent since most of the people that would be using it are likely using it
in a more enterprise/work -related way and thus might lose track of
responding.  (At least that's what happened with me as I follow the blog
and meant to actually respond and just... forgot).  But, all that aside, it
is very much appreciated as I was close to just implementing some type of
internal build job to try and take the fedora source package and try and
build it for my company, but obviously it just much much more efficient and
easier if there is a COPR -based repo to sync from (which is what I
currently do).

**Comment by czanik** [comment]
@mexigabacho update is ready: https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng-amazon23/builds/ I did not change much: just updated to 4.12 and added --enable-slog to configure, as I was lazy to edit the file list :-) As a next step I might disable Java and slog to be in sync with the rest of the packages.

Before I make an announcement (blog), let me know if everything works as expected.
